packages/file_selector/file_selector/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.8.3+1
+* Improve API docs and examples
```

Comment:
Please follow the CHANGELOG style guide linked from the PR checklist.

Reply:
This is not fully resolved; it should be "Improves", not "Improve", as shown in the style guide.